### PR TITLE
IGP13

### PR DESCRIPTION
**Pull request by thrilok209**

### CODE REVIEW

File: contracts/payloads/IGP13/PayloadIGP13.sol [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+    ILite public constant LITE =
+        ILite(0xA0D3707c569ff8C87FA923d3823eC5D81c98Be78);
+
+    address public constant WITHDRAWALS_MODULE =
+        0x6A64A3E0af38279ac7455c85b2C683f5621cE2e7;
+    address public constant FLUID_STETH_MODULE =
+        address(0); // TODO
+
+    address public constant DUMMY_IMPLEMENTATION =
```
Comment (KaymasJain) [inline contracts/payloads/IGP13/PayloadIGP13.sol:137]: This will also be a TODO

File: contracts/payloads/IGP13/PayloadIGP13.sol [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+        0x3AC94ba5465BF3429fE237468EaBa5aa20072d61;
+
+    address public constant VAULT_WEETH_WSTETH = address(0); // TODO
```
Comment (KaymasJain) [inline contracts/payloads/IGP13/PayloadIGP13.sol:140]: https://etherscan.io/address/0x40D9b8417E6E1DcD358f04E3328bCEd061018A82#readContract

File: contracts/payloads/IGP13/PayloadIGP13.sol [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+        IFluidVaultT1(VAULT_WEETH_WSTETH).updateOracle(VAULT_WEETH_WSTETH_ORACLE);
+    }
+
+    /***********************************|
+    |          Function Signatures      |
+    |__________________________________*/
+    
+    function withdrawalsSigs() public pure returns (bytes4[] memory sigs_) {
+        sigs_ = new bytes4[](3);
+        sigs_[0] = bytes4(
```
Comment (KaymasJain) [inline contracts/payloads/IGP13/PayloadIGP13.sol:230]: We can also remove this sig.

File: contracts/payloads/IGP13/PayloadIGP13.sol [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+    ) external;
+
+    /// @notice updates the allowed rebalancer to `newRebalancer_`.
+    function updateRebalancer(address newRebalancer_) external;
+
+    /// @notice updates the supply rate magnifier to `supplyRateMagnifier_`. Input in 1e2 (1% = 100, 100% = 10_000).
+    function updateSupplyRateMagnifier(uint supplyRateMagnifier_) external;
+}
+
+contract PayloadIGP8 {
```
Comment (bergben) [inline contracts/payloads/IGP13/PayloadIGP13.sol:110]: nit: update contract name? 
```suggestion
contract PayloadIGP13 {
```

**Review by bergben — APPROVED** [review]

**Pull request merged**